src/Send/SendScreen/Fee.tsx

```diff
@@ -6,14 +6,15 @@ import {Text} from '../../components'
 import {getDefaultAssetByNetworkId} from '../../legacy/config'
 import {formatTokenWithSymbol} from '../../legacy/format'
 import {useSelectedWallet} from '../../SelectedWallet'
+import {Quantity} from '../../yoroi-wallets/types'
 import {useStrings} from './strings'
 
-export const Fee = ({fee}: {fee: BigNumber | null}) => {
+export const Fee = ({fee}: {fee: Quantity | null}) => {
   const strings = useStrings()
   const wallet = useSelectedWallet()
   const defaultAsset = getDefaultAssetByNetworkId(wallet.networkId)
 
-  const value = fee ? formatTokenWithSymbol(fee, defaultAsset) : strings.feeNotAvailable
+  const value = fee !== null ? formatTokenWithSymbol(new BigNumber(fee), defaultAsset) : strings.feeNotAvailable
```

Comment:
check for truthy should be sufficient

Reply:
`Quantity` is a string, truthy checks were disabled for strings.

Reply:
`Quantity` cannot be an empty string

Reply:
then, if we don't update the the lint rules, we will have to check against `!== null`